code/modules/spells/targeted/disorient.dm

```diff
@@ -19,4 +20,29 @@
 	compatible_mobs = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
 	spell_flags = WAIT_FOR_CLICK
 
-	hud_state = "wiz_disorient"
\ No newline at end of file
+	hud_state = "wiz_disorient"
+	var/empowered = 0
```

Comment:
Why create a snowflake var, instead of checking `spell_levels[Sp_POWER] >= 1` ?

Reply:
because a lot of other things do it and there's virtually no difference

Reply:
consistency with badness isn't good though